scripts/tests/api_compare/docker-compose.yml

```diff
@@ -2,70 +2,186 @@
+      - FULLNODE_API_INFO=/dns/forest/tcp/${FOREST_RPC_PORT}/http
+    entrypoint: [ "/bin/bash", "-c" ]
     command:
       - |
         set -euxo pipefail
-        # if there are some files in the data directory, then we don't need to fetch the snapshot
-        if [ "$$(ls -A /data/snapshot/)" ]; then
-          echo "Snapshot already fetched"
-        else
-          mkdir -p /data/snapshot
-          forest-tool snapshot fetch  --chain ${CHAIN} --directory /data/snapshot/
-        fi
-        # copy proofs to the shared volume, if they are not already there
-        if [ "$$(ls -A /proofs/)" ]; then
-          cp -r ${FIL_PROOFS_PARAMETER_CACHE}/* /proofs/
-        fi
-  forest:
+        forest-cli sync wait
+        forest-cli snapshot export -o "$SNAPSHOT_PATH" -d=900 --include-message-receipts
+  api-serve:
     depends_on:
-      init:
+      snapshot-export:
         condition: service_completed_successfully
-    image: ${FOREST_IMAGE}
+    build:
+      context: ../../../.
+      dockerfile: ${FOREST_DOCKERFILE_OVERRIDE:-Dockerfile}
     volumes:
       - node-data:/data
       - filecoin-proofs:${FIL_PROOFS_PARAMETER_CACHE}
     networks:
       - api-tests
     environment:
       - FIL_PROOFS_PARAMETER_CACHE=${FIL_PROOFS_PARAMETER_CACHE}
-    entrypoint: ["/bin/bash", "-c" ]
+    entrypoint: [ "/bin/bash", "-c" ]
     command:
       - |
         set -euxo pipefail
-        forest --import-snapshot /data/snapshot/* --encrypt-keystore false --chain ${CHAIN} --rpc-address 0.0.0.0:${FOREST_RPC_PORT}
+        forest-tool api serve "$SNAPSHOT_PATH" \
+          --chain calibnet --port ${FOREST_OFFLINE_RPC_PORT}
   lotus:
     depends_on:
-      init:
+      snapshot-export:
         condition: service_completed_successfully
     image: ${LOTUS_IMAGE}
     volumes:
       - node-data:/data
+      - lotus-data:/var/lib/lotus
       - filecoin-proofs:${FIL_PROOFS_PARAMETER_CACHE}
     networks:
       - api-tests
     environment:
       - FIL_PROOFS_PARAMETER_CACHE=${FIL_PROOFS_PARAMETER_CACHE}
       - LOTUS_API_LISTENADDRESS=/ip4/0.0.0.0/tcp/${LOTUS_RPC_PORT}/http
       - LOTUS_FEVM_ENABLEETHRPC=1
-    entrypoint: ["/bin/bash", "-c" ]
+      - LOTUS_CHAINSTORE_ENABLESPLITSTORE=false
+      - FULLNODE_API_INFO=/dns/lotus/tcp/${LOTUS_RPC_PORT}/http
+    entrypoint: [ "/bin/bash", "-c" ]
+    command:
+      - |
+        set -euxo pipefail
+        lotus daemon --import-snapshot "$SNAPSHOT_PATH" --remove-existing-chain
+  lotus-sync-wait:
+    depends_on:
+      lotus:
+        condition: service_started
+    image: ${LOTUS_IMAGE}
+    volumes:
+      - node-data:/data
+      - lotus-data:/var/lib/lotus
+      - filecoin-proofs:${FIL_PROOFS_PARAMETER_CACHE}
+    networks:
+      - api-tests
+    environment:
+      - FULLNODE_API_INFO=/dns/lotus/tcp/${LOTUS_RPC_PORT}/http
+    entrypoint: [ "/bin/bash", "-c" ]
+    command:
+      - |
+        set -euxo pipefail
+        until lotus sync wait; do
+            sleep 5
+        done
```

Comment:
Why not `lotus sync wait`?

Reply:
lotus does not start RPC server until the snapshot is fully imported

Reply:
But I guess I can use the most pulled `alpine` or `busybox` image or this

Reply:
Let's add a comment to help future readers.

Reply:
This is a comment line in volume defination section, should it be sufficient?

```yaml
volumes:
  filecoin-proofs:
  node-data:
  # mount this to /var/lib/lotus to avoid creating random volumes
  lotus-data:
```